### ISSUE TITLE
Design + Content: Add warning messages for certain actions in the admin portal

### ISSUE THREAD

**Issue by kristinacyin** [cisagov/manage.get.gov#1473]
### Story

As an analyst
I want to receive warning messages when I put a domain on hold, mark it as ineligible, or delete a domain 
so that I do not accidently make a major change unintentionally

### Acceptance Criteria

- [ ] Identify statuses/actions in the admin portal that should have a warning (so far: On Hold, Ineligible, Remove from registry)
- [ ] Content is drafted for the warning messages
- [ ] Work with engineer(s) to ideate agreed upon approach
- [ ] Link design approach to dev ticket for implementation (#1530 )

### Additional Context

[Slack thread for context](https://gsa-tts.slack.com/archives/C05BDEA3C11/p1702312238745639)

### Issue Links
Blocks: #1530 (dev ticket)

_No response_
